go/vt/vtgate/executor.go

```diff
@@ -1596,10 +1596,10 @@ func (e *Executor) handlePrepare(ctx context.Context, safeSession *SafeSession,
 
 // ExecuteMultiShard implements the IExecutor interface
 func (e *Executor) ExecuteMultiShard(ctx context.Context, rss []*srvtopo.ResolvedShard, queries []*querypb.BoundQuery, tabletType topodatapb.TabletType, session *SafeSession, notInTransaction bool, autocommit bool) (qr *sqltypes.Result, errs []error) {
-	return e.scatterConn.ExecuteMultiShard(ctx, rss, queries, tabletType, session, notInTransaction, autocommit)
+	return e.scatterConn.ExecuteMultiShard(ctx, rss, queries, session, notInTransaction, autocommit)
```

Comment:
Looks like you can drop the tabletType from the outer function also. Is there a reason to keep it?

Reply:
Done